### DISCUSSION TITLE
How many .gbk files can big-scape process?

### DISCUSSION THREAD

**Discussion by chingciripit** [medema-group/BiG-SCAPE#176] [General]
(i.e, what is the maximum number of .gbk files that big-scape can process?)

**Comment by jorgecnavarrom** [comment]
It depends only on the machine's resources, but it is unfortunately not possible to calculate it directly. For the publication, we did about 75k BGCs (which we ran in parts; first the domain annotation, then sets of biosynthetic classes)

There will be a more extensive benchmarking for BiG-SCAPE 2

**Comment by chingciripit** [comment]
Thanks, Jorge. I always have issues with IndexError: list index out of range so I was wondering if I input too many .gbk files. 

The BiG-SCAPE always stops after this analysis. Do you know what the issue is and how to troubleshoot this?

Thank you.

  Mix (5182 BGCs)
  Calculating all pairwise distances
generate_network took 537.680 seconds
  Writing output files
  Calling Gene Cluster Families
  Cutoff: 0.3

![image](https://github.com/user-attachments/assets/3fedda6d-3693-41ad-997b-f83b14518074)

**Comment by jorgecnavarrom** [comment]
It's possible that the problem arises from a newer version of scikit-learn than the one we used. Could you double check if you have v0.19.2 (and install this version if not)?
